libs/partners/unstructured/pyproject.toml

```diff
@@ -0,0 +1,87 @@
+[tool.poetry]
+name = "langchain-unstructured"
+version = "0.1.0"
+description = "An integration package connecting Unstructured and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/unstructured"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain-core = "^0.2.0"
```

Comment:
make this more recent

Reply:
This file was generated when I initialized the partner package with `langchain-cli integration new` I believe. I don't have much experience with pyproject.toml files, but are you saying I should update this part manually?